views/default/object/plugin_project/search.php

```diff
@@ -10,6 +10,14 @@
 	'encode_text' => TRUE,
 ));
 
+$img_files = $project->getScreenshots();
+
+if ($img_files) {
+	$thumb = get_entity($img_files[0]->thumbnail_guid);
+	$src = elgg_get_site_url() . "plugins/icon/{$img_files[0]->getGUID()}/icon.jpg";
+	$preview_img = "<a href=\"$project->getURL()\" class=\"plugin_preview\"><img src=\"$src\" alt=\"$project->title\" title=\"$project->title\" width=\"150\"/></a>";

```

Comment:
i was just following the format of the other elements in the plugin. do you want me to change it?